engine/api/src/main/scala/pl/touk/nussknacker/engine/api/typed/typing.scala

```diff
@@ -94,11 +100,14 @@ object typing {
       case many => many.map(_.display).mkString(" | ")
     }
 
+    override def javaClassName: String = ???
   }
 
   //TODO: make sure parameter list has right size - can be filled with Unknown if needed
   case class TypedClass private[typing] (klass: Class[_], params: List[TypingResult]) extends SingleTypingResult {
 
+    override def javaClassName :String = ReflectUtils.fixedClassSimpleNameWithoutParentModule(ClassUtils.primitiveToWrapper(klass))
```

Comment:
we can't use simple name here because AFAIK spel expects full name, right? why tests doesn't failed?

Reply:
Made some changes, please take another look